test/presenters/return-requirements/add-note.presenter.test.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const AddNotePresenter = require('../../../app/presenters/return-requirements/add-note.presenter.js')
+
+describe('Add Note presenter', () => {
+  let session
+
+  beforeEach(() => {
+    session = {
+      id: 'f1288f6c-8503-4dc1-b114-75c408a14bd0',
+      data: {
+        licence: {
+          id: 'ea53bfc6-740d-46c5-9558-fc8cabfc6c1f',
+          licenceRef: '01/123',
+          licenceHolder: 'Jane Doe'
+        },
+        note: 'Note attached to return requirement'
```

Comment:
With the changes made in `SubmitAddNoteService` this is no longer how the note will be stored on the session.

The test and the presenter need to be updated to expect

```suggestion
        note: {
          content:  'Note attached to return requirement',
          userEmail: 'jo@example.com'
        }
```